packages/strapi-admin/admin/src/themes/colors.js

```diff
@@ -35,6 +35,27 @@ const colors = {
     'blue-dark': '#151c2e',
     blue: '#0097f7',
   },
+  filters: {
```

Comment:
I think that the theme should be "features agnostic". To keep just one version of each value. Also, some values can be used in several use cases/contexts. For example, if we need to use the color `filters.button.hover.background` outside a filter, we should still get the `filters.button.hover.background`. Also, this prevents us from having duplicates colors.

Reply:
Yes you're right. I allowed myself to do it because this file already contains this kind of structure and I thought it was something we wanted for the theme but I will change it because I agree.